frontend/packages/dev-console/src/components/topology/components/nodes/AlertSeverityIcon.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
```

Comment:
Do we need this under topology/nodes?

Reply:
I have moved this component to `console-shared/src/components/alerts/`